tests/DefaultCacheTest.php

```diff
@@ -3,6 +3,7 @@
 namespace ipinfo\ipinfo\tests;
 
 use ipinfo\ipinfo\cache\DefaultCache;
+use Symfony\Component\Cache\Adapter\FilesystemAdapter;
```

Comment:
Why is this imported here now?

Reply:
Because we have initialized it internally here.
`$this->cache = new FilesystemAdapter();` in constructor.

Reply:
Removed.